src/sandwich-menu/containers/hooks/use-default-sandwich-menu.js

```diff
@@ -0,0 +1,7 @@
+import {useState} from "react";
+
+export const useDefaultSandwichMenu = () => {
+    const [data, setData] = useState([])
```

Comment:
Do we ever use these `data` and `setData`?

Reply:
Done